content/get-started/quickstart/hello-world.md

```diff
@@ -116,18 +117,18 @@ You can even open pull requests in your own repository and merge them yourself.
    ![diff example](/assets/images/help/repository/diffs.png)
 
 5. Click **Create pull request**.
-6. Give your pull request a title and write a brief description of your changes. You can include emojis and drag and drop images and gifs.
+6. Give your pull request a title and write a brief description of your changes. You can include emojis and drag and drop images and gifs. To the right of your title and description, you can assign reviewers or other project members to check out our request by clicking the gear symbol by each option. You can also choose to add labels to provide a bit more organization. Since this is a simple walk-through just for you, there's no need to include any of those options now, but they are there for you in the future!
 7. Click **Create pull request**.
 
 Your collaborators can now review your edits and make suggestions.
 
 ## Merging your pull request
 
-In this final step, you will merge your `readme-edits` branch into the `main` branch.
+In this final step, you will merge your `readme-edits` branch into the `main` branch. After you created the pull request in the above step, you will be brought to a page that shows you any conflicts your code may have with the branch you are merging to - the `main` branch. You can walk through those changes and conflicts and address them with your team members in the comments. For our purposes in this walk-through, there are no conflicts, so you can go ahead and merge your branch into the main branch. You’ll have one more chance to change or edit your commit message.
```

Comment:
```suggestion
In this final step, you will merge your `readme-edits` branch into the `main` branch.  After you merge your pull request, the changes on your `readme-edits` branch will be incorporated into `main`.

Sometimes, a pull request may introduce changes to code that conflict with the existing code on `main`. If there are any conflicts, {% data variables.product.product_name %} will alert you about the conflicting code and prevent merging until the conflicts are resolved. You can make a commit that resolves the conflicts or use comments in the pull request to discuss the conflicts with your team members.

In this walk-through, you should not have any conflicts, so you are ready to merge your branch into the main branch.
```

I think we should separate the information about merge conflicts. It's important info, but might be a lot for someone new to understand at once. The main thing I think we need to emphasize here is that merging is what brings your changes from a branch into `main`.